Element with id All Openhttps://github.com/microsoft/vscode/pull/260521 is already registered

I create a PR using the extension. Then the tree view refreshed twice automatically which then ended up showing the notification error: `Element with id All Openhttps://github.com/microsoft/vscode/pull/260521 is already registered`. 

In the console I see

```
notificationsAlerts.ts:40 Error: Element with id All Openhttps://github.com/microsoft/vscode/pull/260521 is already registered
    at s5.eb (file:///c:/Users/besimmonds/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:196:82731)
    at file:///c:/Users/besimmonds/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:196:81579
    at Array.map (<anonymous>)
    at s5.Z (file:///c:/Users/besimmonds/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:196:81561)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at s5.getChildren (file:///c:/Users/besimmonds/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:196:77878)
    at pG.$getChildren (file:///c:/Users/besimmonds/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:196:74493)
```

This lead to the entire tree view to not show any PRs anymore

<img width="368" height="388" alt="Image" src="https://github.com/user-attachments/assets/7a9ad135-7546-4ad9-aa77-3d0345816e0a" />

--- a/src/vs/workbench/api/common/extHostTreeViews.ts
+++ b/src/vs/workbench/api/common/extHostTreeViews.ts
@@ -413,6 +413,8 @@ class ExtHostTreeView<T> extends Disposable {
 					return this._refresh(elements).then(() => {
 						this._clearNodes(childrenToClear);
 						return _promiseCallback();
+					}).catch(_ => {
+						this._clearNodes(childrenToClear);
 					});
 				});
 			}

